### DISCUSSION TITLE
Sharing?

### DISCUSSION THREAD

**Discussion by berot3** [wallabag/ios-app#421] [Q&A]
Hey

Wanted to give wallabag.it with the ios-app a chance (omnivore-refugee here).

But i cant see the wallabag-icon when sharing a site from safari. Am i doing something wrong? Never had such a issue with an app before...

**Comment by bourvill** [comment]
Hello, have you tried going to the end of the list in the menu that allows you to see all the apps?

**Reply by gabeklavans** [reply]
I can also say that I had to open the menu that lets me see all apps and wallabag was at the end of that. I'm not sure if there's a way to have wallabag show up in the regular list by default, but that would probably be nice here.
